### ISSUE TITLE
Typo in Lambda layer

### ISSUE THREAD

**Issue by Witiko** [akarshzingade/image-similarity-deep-ranking#29]
Line 28 of `deepRanking.py` contains the following:
```python
x = Lambda(lambda x_: K.l2_normalize(x,axis=1))(x)
```
Should not we be passing `x_` rather than `x` to `K.l2_normalize`?

**Comment by elenaterenzi** [comment]
yep, using that code we were receiving many strange errors during training phase. removing the underscore solved the problem for us
